builds/android/app/src/main/res/values/strings.xml

```diff
@@ -104,4 +104,8 @@ Please tell us in detail what went wrong.\n
 	<string name="key_enter">Enter (Z)</string>
 	<string name="key_cancel">Cancel (ESC and X)</string>
 	<string name="key_shift">Shift</string>
+
+    <!--GameBrowserAPI15-->
+    <string name="navigation_drawer_open">Open navigation drawer</string>
+    <string name="navigation_drawer_close">Close navigation drawer</string>
```

Comment:
Oh, I didn't saw the problem because it's indented with 4 space, and the other with tab (no difference in my IDE).
Because 4 space is the Java standard (which is stupid imho), I will try to keep this way to indent.